packages/lesswrong/lib/collections/posts/views.js

```diff
@@ -20,9 +20,7 @@ Posts.addDefaultView(terms => {
       draft: {$in: [false,null]},
       isFuture: {$in: [false,null]}, // match both false and undefined
       unlisted: {$in: [false,null]},
-      meta: {$in: [false,null]},
-      groupId: {$exists: false},
-      isEvent: {$in: [false,null]},
```

Comment:
This causes events, group posts, and meta posts to appear in a bunch of places they didn't before.

Reply:
This was motivated by reasoning:

a) on User Profiles at least, we want all three of these things

b) on every other place Oli and I could think of except for frontpage, these were largely irrelevant. 

Recent Discussion wants all those things. Frontpage only includes things we manually put there. 

Curated only includes things we manually put there. 

All posts should include meta. (It shouldn't include groupId or event posts, but it's the only view where that was true)